providers/implementations/kdfs/snmpkdf.c

```diff
@@ -265,18 +265,21 @@ const OSSL_DISPATCH ossl_kdf_snmpkdf_functions[] = {
  *
  * Shared_key = SHA-1(Derived_password || snmpEngineID || Derived_password).
  *
+ * Input:
```

Comment:
This change does not belong in the PR.
It looks like a bug fix, and it should also be tested.